CMakeLists.txt

```diff
@@ -30,6 +31,34 @@ endif()
 
 cmake_minimum_required(VERSION 3.16)
 
+# search for Vulkan SDK
+find_package(Vulkan)
+
+if(VULKAN_FOUND)
+    if(NOT Vulkan_dxc_EXECUTABLE)
+        if(WIN32)
+            find_program(Vulkan_dxc_EXECUTABLE
+                        NAMES dxc
+                        HINTS
+                        "$ENV{VULKAN_SDK}/Bin"
+                        )
+        else()
+            find_program(Vulkan_dxc_EXECUTABLE
+                        NAMES dxc
+                        HINTS
+                        "$ENV{VULKAN_SDK}/bin"
+                        )
+        endif()
```

Comment:
The two branches are very close!
Maybe something like this is easier to grasp:
```
if (WIN32)
    set( BinName Bin )
else()
    set( BinName bin )
endif()
find_program(... "$ENV{VULKAN_SDK}/${BinName})
```

Reply:
Thanks for suggestion @jherico, I included both hints in `find_program` call.